lib/service/connect.go

```diff
@@ -1044,12 +1044,34 @@ func (process *TeleportProcess) rotate(conn *Connector, localState auth.StateV2,
 // falls back to trying to connect to the Auth Server through the proxy.
```

Comment:
Do we need to update the godoc here? For config v3 we should not be trying to connect to both auth and proxy.